<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BedMicroscopeWidget</class>
 <widget class="QWidget" name="BedMicroscopeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="4">
    <widget class="Display" name="imageDisplay" native="true"/>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="0">
            <widget class="QLabel" name="saveFolderLabel">
             <property name="text">
              <string>Select Save Location</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="setSaveFolderButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Save Folder:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="1">
            <widget class="QSpinBox" name="bedIDSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Bed ID</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Save Image</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item row="2" column="0" rowspan="2" colspan="2">
          <widget class="QPushButton" name="connectButton">
           <property name="text">
            <string>Connect to Microscope Camera</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Camera ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QComboBox" name="cameraIDComboBox">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item row="5" column="5">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>mm</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QDoubleSpinBox" name="ySpacingSpinBox">
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Num Y</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Y Spacing</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QDoubleSpinBox" name="yStartPositionSpinBox"/>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="xStartPositionSpinBox"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Num X</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>mm</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>mm</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>mm</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="6">
          <widget class="QPushButton" name="setStartPositionButton">
           <property name="text">
            <string>Set Current Position as Start Position</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Start Position y</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Start Position x</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="6">
          <widget class="QPushButton" name="captureImagesButton">
           <property name="text">
            <string>Capture Images</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="xSpacingSpinBox">
           <property name="value">
            <double>4.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QSpinBox" name="numYSpinBox">
           <property name="value">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>X Spacing</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="numXSpinBox">
           <property name="value">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="6">
          <widget class="QPushButton" name="moveToStartButton">
           <property name="text">
            <string>Move to Start Position</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Display</class>
   <extends>QWidget</extends>
   <header>display.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cameraIDComboBox</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>bedIDSpinBox</tabstop>
  <tabstop>xSpacingSpinBox</tabstop>
  <tabstop>ySpacingSpinBox</tabstop>
  <tabstop>numXSpinBox</tabstop>
  <tabstop>numYSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
